parser/src/AST/Module.hs

```diff
@@ -60,6 +60,6 @@ data UserImport
 
 data ImportMethod = ImportMethod
     { alias :: Maybe (Comments, PreCommented String)
-    , exposedVars :: (Comments, PreCommented (Var.Listing Var.Value))
+    , exposedVars :: (Comments, PreCommented (Var.Listing Var.Value), Bool)
```

Comment:
This is hardly unique to this PR but I wish there was a way to document that the bool signifies whether the import is multiline or not.